packages/nibijs/src/msg/spot.ts

```diff
@@ -86,21 +86,35 @@ export class SpotMsgFactory {
   static joinPool(msg: MsgJoinPool): TxMessage {
```

Comment:
msg could be destructured here; `{poolId, sender, tokensIn, useAllCoins}: MsgJoinPool`

Then you could have

```      value: MsgJoinPool.fromPartial({
        poolId: Number(msg.poolId),
        sender,
        tokensIn,
        useAllCoins,
      }),```

Same with all changes below